### PR TITLE
QAP-46 New Balance-1 e2e buying running shoes

### PR DESCRIPTION
**Pull request by YevheniiBond**
Guys, waiting for your attention ^_^

### CODE REVIEW

File: src/test/java/YevheniiBondarenko/Tests/Pages/NewBalanceHelper.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package YevheniiBondarenko.Tests.Pages;
+
+import org.openqa.selenium.WebDriver;
+
+public class NewBalanceHelper extends NewBalanceWebElement{
+    public NewBalanceHelper(WebDriver driver) {
+        super(driver);
+    }
```
Comment (Tsibatyuk) [inline src/test/java/YevheniiBondarenko/Tests/Pages/NewBalanceHelper.java:8]: add one line betwen methods

Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Pages/NewBalanceHelper.java:8]: Wait a moment

**Review by Tsibatyuk — APPROVED** [review]

File: src/test/java/YevheniiBondarenko/Tests/Pages/NewBalanceHelper.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package YevheniiBondarenko.Tests.Pages;
```
Comment (Safron1ta2) [inline src/test/java/YevheniiBondarenko/Tests/Pages/NewBalanceHelper.java:1]: Enter after pacage (ctrl+alt+o)

File: src/test/java/YevheniiBondarenko/Tests/Tests/NewBalance1E2EByingRunningShoes.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        newBalanceWebElement.shoes().click();
+        newBalanceWebElement.kindOfShoes().get(0).click();
+        newBalanceWebElement.shoesForCityRoad().click();
+        newBalanceWebElement.genderMan().click();
+        newBalanceWebElement.listOfSize().get(1).click();
+        newBalanceWebElement.myChoice().click();
+        newBalanceWebElement.sizeBeforeCart().click();
+        newBalanceWebElement.addToCart().click();
+
+        sleep(1);
```
Comment (AndriiMyronenko418) [inline src/test/java/YevheniiBondarenko/Tests/Tests/NewBalance1E2EByingRunningShoes.java:26]: Why do we need sleep() here?
If we really need it, then please comment on it in your code.

**Review by AndriiMyronenko418 — CHANGES_REQUESTED** [review]

File: src/test/java/YevheniiBondarenko/Tests/Tests/NewBalance1E2EByingRunningShoes.java (hunk 3 above)
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Tests/NewBalance1E2EByingRunningShoes.java:26]: Sure! I tried to do this part without 'sleep', but it's all the time falling. Thank for your feedback, next time I'll leave a //comment!

File: src/test/java/YevheniiBondarenko/Tests/Pages/NewBalanceHelper.java (hunk 2 above)
Comment (YevheniiBond) [reply src/test/java/YevheniiBondarenko/Tests/Pages/NewBalanceHelper.java:1]: Ok! Thanks for your feedback.

**Review by Safron1ta2 — APPROVED** [review]

**Review by AndriiMyronenko418 — APPROVED** [review]

**Pull request merged**